src/fides/api/models/openid_provider.py

```diff
@@ -0,0 +1,31 @@
+import enum
+
+from sqlalchemy import Boolean, Column
+from sqlalchemy import Enum as EnumColumn
+from sqlalchemy import String
+from sqlalchemy.orm import relationship
+
+from fides.api.db.base_class import Base
+from fides.api.models.audit_log import AuditLog
+
+
+class ProviderEnum(enum.Enum):
+    google = "google"
+
+
+class OpenIDProvider(Base):
+    """The DB ORM model for OpenIDProvider."""
+
+    provider = Column(EnumColumn(ProviderEnum), unique=True, index=True)
+    client_id = Column(String, nullable=False)
+    client_secret = Column(String, nullable=False)
```

Comment:
We should encrypt these values for safety, here's an example usage of the `StringEncryptedType`

```
class ConsentIdentitiesMixin:
    """Encrypted and hashed identities for consent reporting and last saved preference retrieval"""

    email = Column(
        StringEncryptedType(
            type_in=String(),
            key=CONFIG.security.app_encryption_key,
            engine=AesGcmEngine,
            padding="pkcs5",
        ),
    )  # Encrypted email
```